pkg/models/domain/cost.go

```diff
@@ -2,14 +2,20 @@ package domain
 
 import "time"
 
+type UsageStats struct {
+	RecordsCount    int64
+	FirstRecordTime *time.Time
+}
+
 type CostComponent struct {
 	Type        string  // compute
 	Value       float64 // 2
 	Unit        string  // machines
 	TotalAmount float64 // Value * Rate
 	Rate        float64 // 0.0042
 	Currency    string  // USD
-	Description string  // "price for 2 x t4g.nano"
+	SKU         string
```

Comment:
What is SKU? Can you please add an example to the model?

Reply:
it's a sku_name column from usage table in databricks.
<img width="1089" height="396" alt="Screenshot 2025-07-29 at 08 22 55" src="https://github.com/user-attachments/assets/a9c6741e-9a18-4ae0-bb8b-f24ac9bed473" />
I will add a comment.